src/components/organism/ShopProducts/OrderPage/OrderPage.js

```diff
@@ -7,6 +9,8 @@ import './OrderPage.scss';
 
 const OrderPage = () => {
   const cartItems = useSelector(state => state.cart.cartItems);
+  const [id] = useState(_uniqueId('prefix-'));
```

Comment:
It's great that we rely on a unique id as keys for React! :) However, the items in store already have their ids, so we do not need to generate an extra id - we can simply use `item.id`. 
Also, there is already a unique id helper in the project (`generateUUID`). It's probably best if we use `generateUUID` or `lodash` for unique ids, not both, to avoid having such duplication in the project.